Source/SessionManager.swift

```diff
@@ -58,10 +58,18 @@ open class SessionManager {
         let acceptEncoding: String = "gzip;q=1.0, compress;q=0.5"
 
         // Accept-Language HTTP Header; see https://tools.ietf.org/html/rfc7231#section-5.3.5
+        #if swift(>=4.0)
+            let acceptLanguage = Locale.preferredLanguages.prefix(6).enumerated().map { enumeratedLanguage in
+                let (index, languageCode) = enumeratedLanguage
+                let quality = 1.0 - (Double(index) * 0.1)
+            return "\(languageCode);q=\(quality)"
+        }.joined(separator: ", ")
+        #else
         let acceptLanguage = Locale.preferredLanguages.prefix(6).enumerated().map { index, languageCode in
             let quality = 1.0 - (Double(index) * 0.1)
             return "\(languageCode);q=\(quality)"
         }.joined(separator: ", ")
+        #endif
```

Comment:
Addressed in [a0337](https://github.com/Alamofire/Alamofire/pull/2163/commits/03dbf047a5fd956e850a135efbf651c390ea0337).